frontend/gradlew

```diff
@@ -28,16 +44,16 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
@harshbafna Can you please explain what was the reason for adding these JVM OPTS values?

Reply:
This file has been updated by `gradle wrapper` command, the official tool for upgrading gradle, and has been checked in as is.

The script updates the following files :

 - gradlew
 - gradlew.bat
 - gradle-wrapper.properties (updates the distribution URL)
 - gradle-wrapper.jar